examples/PowerElectronics/DistributedGeneratorTest/DGTest.cpp

```diff
@@ -49,12 +49,12 @@ int main(int argc, char const* argv[])
 
   dg->evaluateResidual();
 
-  std::cout << "Output: {";
-  for (double i : dg->getResidual())
-  {
-    printf("%e ,", i);
-  }
-  std::cout << "}\n";
+  // std::cout << "Output: {";
```

Comment:
Consider adding a verbosity flag and a conditional to get to this code. Or remove completely.

Reply:
Need to add CLI utilities to GridKit first.